package.json

```diff
@@ -9,8 +9,11 @@
     "type": "git",
     "url": "git://github.com/bigcommerce/stencil-utils.git"
   },
+  "scripts": {
+    "test": "npx grunt eslint && npx grunt karma"
+  },
   "dependencies": {
-    "eventemitter2": "^0.4.14",
+    "eventemitter2": "^6.4.3",
```

Comment:
This is a major bump. Do we have any tests in place to confirm there were no breaking change or how it was manually tested ?

Reply:
The tests are under https://github.com/bigcommerce/stencil-utils/tree/master/src/test-unit
Also linked the package into cornerstone, which uses stencil-utils, and run tests over there

Reply:
Agreed we have some unit tests but I don’t believe we have for events which makes me wonder how confidently we can make this change. I will have to look if cornerstone has any tests related to it.

At the very least we should manually tests the flows on cornerstone where the events are emitted to make sure nothing is broken.

Reply:
Did we also evaluate https://github.com/primus/eventemitter3?

Reply:
@bookernath Initially, I haven't, just looked at the number of start. Libraries provide pretty much same functionality and we are using only 3 methods: subscribe to event, unsubscribe an event and emit an event. The only benefit I'm seeing from eventemitter3 is less package size, so I replaced the eventemitter2 to 3.